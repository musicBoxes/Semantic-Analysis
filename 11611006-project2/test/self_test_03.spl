int invalid_operation_test(){
	int a = 1.0;
	int b = 0;
	float c = 1.0;
	char d = 'c';
	
	b = c;
	c = b;
	d = d + 1;
}