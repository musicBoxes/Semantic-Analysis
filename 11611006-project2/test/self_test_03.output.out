Error type 5 at Line 2: unmatching type on both sides of assignment
Error type 5 at Line 7: unmatching type on both sides of assignment
Error type 5 at Line 8: unmatching type on both sides of assignment
Error type 7 at Line 9: Invalid operation 'PLUS' on non-number variables
Error type 5 at Line 9: unmatching type on both sides of assignment
